src/unity/python/turicreate/visualization/__init__.py

```diff
@@ -3,6 +3,121 @@
 #
 # Use of this source code is governed by a BSD-3-clause license that can
 # be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+#
+# Turi Create Visualization allows you to create Plot objects and 
+# manipulate them using various functionalities, including
+# saving them (.save) and viewing them (.show). Moreover, Turi Create 
+# Visualization also allows the user to call `show` on data sources like 
+# SFrames and SArrays, and provides functionality to create independent plots 
+# like scatter plots, categorical heatmaps, box plots, heatmaps, columnwise 
+# summaries, and item frequency plots.
+#
+# ==========
+# class Plot
+# ==========
+#
+# 	An immutable object representation of a visualization.
+#
+#   Notes
+#   -----
+#   - A plot object is returned via the SFrame's or SArray's .plot() method
+#
+#   Examples
+#   --------
+#   Suppose 'plt' is an Plot Object
+#
+#   We can view it using:
+#   >>> plt.show()
+#
+#   We can save it using:
+#   >>> plt.save('vega_spec.json')
+#
+#   We can also save the vega representation of the plot without data:
+#   >>> plt.save('vega_spec.json', False)
+#
+#   We can save the plot as a PNG/SVG using:
+#   >>> plt.save('test.png')
+#   >>> plt.save('test.svg')
+#
+# ====
```

Comment:
This won't render correctly at all. Can you check how this renders.

Reply:
Where would these render though? There are separate, correctly formatted docs for the API docs, next to each individual function. This commit was to make the response to `help(tc.visualization)` more useful (from the command line)

Reply:
The help text and API docs should be the same (and come from the same place).

Reply:
I think you'll want to put the help/API text in triple quotes at the top of the file (perhaps above the license, if that's what is needed to get it to show up correctly).